airflow/settings.py

```diff
@@ -90,7 +90,7 @@ def timing(cls, stat, dt):
 ____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
 ___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
  _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
- """
+ """  # noqa: W605
```

Comment:
The same to @feng-tao 's question: why do we need `# noqa: W605` here? I tried to run flake8 test on the original code and no issue was raised.

Reply:
The W605 error wasn't checked due to a bug prior to flake8 3.6. Are you running the latest flake8? When I run flake8 without ignoring W605, I get:

```
airflow/settings.py:93:-121: W605 invalid escape sequence '\_'
airflow/settings.py:93:-26: W605 invalid escape sequence '\_'
```

We could also make the `HEADER` variable a rawstring. WDYT?

Reply:
Get it. Overall it looks good to me (pinning flake8 to 3.5.0 was a temporary fix). Thanks!

The only thing I'm still considering is how the string chunks should be handled, like the `HEADER` here. I would like to leave this to the committers to decide.

Reply:
Yes, a raw string would be the way to go here :)

Reply:
I took another look, and switching to a raw string would escape the backslash on the first line, which we don't want. I refactored slightly to avoid literal backslashes and to make sure the ascii text lines up.